brayovsky/pivotaly
# Refresh story and cycle time after project edit 165999004

#### What does this PR do?
- Refreshes story info view and member cycle view after changing token or project
- Links a story after changing token or project
- Adds refresh member cycle data command

#### How should this be manually tested?
When one:
- Opens extension
- Changes project or token successfully

Then:
- One should be asked to pick a story
- Story information view should update accordingly
- Member cycle time view should update accordingly

#### Any background context you want to provide?
n/a

#### Screenshots (if appropriate)
n/a
#### Questions:
n/a
